CMakeLists.txt

```diff
@@ -1,15 +1,19 @@
 cmake_minimum_required(VERSION 3.15)
-project (crow_all)
+project (crow_all LANGUAGES CXX)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
 
+# Set required C++ Standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
+# Generate the compile_commands.json file
+set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
```

Comment:
This is typically set by tools when it's required. But it has no negative impact otherwise

Reply:
@madduci you can delete this set.